### DISCUSSION TITLE
How do you style search box in searchable select box?

### DISCUSSION THREAD

**Discussion by allw1994** [mdbootstrap/TW-Elements#1428] [Support from community] [answered]
I'm trying to alter the theming of the searchable select box because the default theming does not match the theming around it.

The docs say to stick the select box in a div and apply theming to the div, but this only seems to affect the "button" part of the select box, not the dropdown.

For instance, the following code:

````
<div class="border rounded-md border-white focus:border-orange-600 focus:border-orange-700">
    <select data-te-select-init data-te-select-filter="true">
        <option value="1">One</option>
        <option value="2">Two</option>
        <option value="3">Three</option>
    </select>
</div>
````
gives the following select box with purple theming and not orange, additionally, the white border seems to be only on top of the default theming:
![image](https://user-images.githubusercontent.com/9084938/221677533-d166e12e-22dd-47c7-bb4d-e180925b9ba5.png)

Any suggestions would be appreciated.

**Comment by Trochonovitz** [comment, marked as answer]
Hi, for customization's options check out [Select's API inTailwind Elements docs](https://tailwind-elements.com/docs/standard/forms/select/#docsTabsAPI). All of classes using by us are fully customizable.
